sherpa-onnx/python/sherpa_onnx/offline_recognizer.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c)  2023 by manyeyes
+from pathlib import Path
+from typing import List
+
+from _sherpa_onnx import (
+    OfflineFeatureExtractorConfig,
+    OfflineRecognizer as _Recognizer,
+    OfflineRecognizerConfig,
+    OfflineStream,
+    OfflineModelConfig,
+    OfflineTransducerModelConfig,
+    OfflineParaformerModelConfig,
+)
+
+
+def _assert_file_exists(f: str):
+    assert Path(f).is_file(), f"{f} does not exist"
+
+
+class OfflineRecognizer(object):
+    """A class for offline speech recognition."""
+
+    @classmethod
+    def from_transducer(
+            cls,
+            encoder: str,
+            decoder: str,
+            joiner: str,
+            tokens: str,
+            num_threads: int,
+            sample_rate: int = 16000,
+            feature_dim: int = 80,
+            decoding_method: str = "greedy_search",
+            debug: bool = False,
+    ):
+        """
+        Please refer to
+        `<https://k2-fsa.github.io/sherpa/onnx/pretrained_models/index.html>`_
+        to download pre-trained models for different languages, e.g., Chinese,
+        English, etc.
+
+        Args:
+          tokens:
+            Path to ``tokens.txt``. Each line in ``tokens.txt`` contains two
+            columns::
+
+                symbol integer_id
+
+          encoder:
+            Path to ``encoder.onnx``.
+          decoder:
+            Path to ``decoder.onnx``.
+          joiner:
+            Path to ``joiner.onnx``.
+          num_threads:
+            Number of threads for neural network computation.
+          sample_rate:
+            Sample rate of the training data used to train the model.
+          feature_dim:
+            Dimension of the feature used to train the model.
+          decoding_method:
+            Valid values are greedy_search, modified_beam_search.
+          debug:
+            True to show debug messages.
+        """
+        self = cls.__new__(cls)
+        model_config = OfflineModelConfig(
+            transducer=OfflineTransducerModelConfig(
+                encoder_filename=encoder,
+                decoder_filename=decoder,
+                joiner_filename=joiner
+            ),
+            paraformer=OfflineParaformerModelConfig(
+                model=""
+            ),
+            tokens=tokens,
+            num_threads=num_threads,
+            debug=debug
+        )
+        decoding_method = decoding_method
```

Comment:
```suggestion
```

Reply:
This line of code is redundant, thank you